staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -56,9 +56,12 @@ var defaultSleepDuration = 10 * time.Second
 
 // TODO: validate impact of copying and investigate hashing
 type eventKey struct {
+	eventType           string
```

Comment:
@wojtek-t any idea why this wasn't kept?  it seems to me that event-sameness does rely on the eventType. Different eventTypes should be different.

Reply:
Seems like a bug to me.
Yes - this should be included.